html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-fg.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<title>document wg color</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-fgcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-linkcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-vlinkcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-alinkcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-bgcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body text="red" linkColor="green" vlink="blue" alink="orange" bgColor="yellow">
+<script>
+test(function() {
+  assert_equals(document.fgColor, document.body.text);
+
```

Comment:
I this test, let’s confirm that each of these is “red”. 
assert_equal(document.fgColor, “red”);

And so on...